cdap-app-fabric/src/main/java/org/apache/twill/yarn/YarnSecureStore.java

```diff
@@ -17,26 +17,44 @@
  */
 package org.apache.twill.yarn;
 
+import com.google.common.base.Throwables;
 import org.apache.hadoop.security.Credentials;
+import org.apache.hadoop.security.UserGroupInformation;
 import org.apache.twill.api.SecureStore;
 
+import java.io.IOException;
+
 /**
  * A {@link SecureStore} for hadoop credentials.
  */
 public final class YarnSecureStore implements SecureStore {
```

Comment:
The `update()` is method is on the `TokenSecureStoreUpdater`, which is a updater, not a `SecureStore`. What have to do with guice here?